paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/lpms/CardLpmTest.kt

```diff
@@ -0,0 +1,39 @@
+package com.stripe.android.paymentelement.confirmation.lpms
+
+import com.stripe.android.model.CardParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+internal class CardLpmTest(
+    private val testType: TestType,
+) : BaseLpmNetworkTest(PaymentMethod.Type.Card) {
+    @Test
+    fun default() = test(
+        testType = testType,
+        createParams = PaymentMethodCreateParams.createCard(
```

Comment:
Thoughts on passing in `ConfirmationHandler.Option` instead?

Reply:
I see. I think this kinda makes sense. The `test` function is a helper, that will eventually create a `ConfirmationHandler.Option`. But we'd need a new `test` helper function for other `ConfirmationHandler.Option` variants.

I think this is fine then 👍